client/Scenes/Game_Main/Interface/Scripts/Inventory.gd

```diff
@@ -34,7 +34,7 @@ func _process(delta):
 				get_node(body_part['ident']).get_node('slot0').button.icon = at
 				
 				if('opened' in body_part['slot0']):
-					if(body_part['slot0']['opened'] == 'yes'):
```

Comment:
this must remain yes or no.

Reply:
Why?  I changed this on the client side too.  It's much more efficient to use solutions built-in to the language rather than rolling your own.  You also lose boolean operations:
* Implicit true/false evaluations `if boolean_value:`, 
* Bitwise operations, 
* Automatic serialization / deserialization
* Readability (most people will expect true/false, not yes/no)

Reply:
Realized a typo, meant server* side.